geniza/corpus/templates/corpus/snippets/document_result.html

```diff
@@ -68,6 +68,22 @@ <h2 class="title">
                     {% endif %}
                 {% endwith %}
 
+                {# translation: keywords in context if any, or excerpt #}
+                {% with document_highlights=highlighting|dict_item:document.id lang=document.translation_language_code %}
+                    <!-- indicate language when possible, or at least change of language (if unknown) for screen readers -->
+                    {% if document_highlights.translation or document.translation %}
+                        <div class="translation" lang="{{ lang }}" dir="ltr">
```

Comment:
Where are you getting language code? this may be an argument for indexing as `transcription_en` etc so you always know what the language is....

Not sure you should assume translation is LTR, I thought there were Hebrew translations — although I'm sure the project team could advise better on this. (Also would be nice to support translations in other LTR languages in future.)

Reply:
I'm indexing the language code as `translation_language_code_s`, but it's actually pretty unstable because it just picks the language code of the last translation it happens to encounter in the loop through Footnotes. Maybe I shouldn't do that!

I had actually intended this to respect RTL/LTR of that language code, similarly to how I did it with the main translation viewer, but must have forgotten. That in itself is quite hacky:

https://github.com/Princeton-CDH/geniza/blob/882b91b084885aac531787dc8966c2fb44964efa/geniza/corpus/templates/corpus/snippets/document_transcription.html#L189

It's just hard-coded checking for a couple of specific RTL languages that are likely to have translations. So at least one thing that I should probably do is add a `direction` field to the `SourceLanguage` model and use that when possible. (Or perhaps even merge that with `LanguageScript` and add `direction` to that model! Any reason why those two are separate?)